documentation/airport-guides/kmia.md

```diff
@@ -0,0 +1,121 @@
+A - Final South (unavailable)
+N - Feeder North (unavailable)
+S - Feeder South (unavailable)
+D - Departure North (unavailable)
+W - Departure South (unavailable)
+F - FLL Final North (unavailable)
+J - FLL Feeder/Final South (unavailable)
+R - FLL North Feeder (unavailable)
+Q - FLL Southwest (unavailable)
+Z - FLL Southeast (unavailable)
+L - FLL North Departure (unavailable)
+G - FLL Satellite (unavailable)
+H - TMB Arrival (unavailable)
+```
+
+### Airports of Interest
+
+#### KMIA (Primary Airport)
+Miami International
+426,474 annual operations
+
+Miami is, of course, the busiest airport being served by Miami TRACON. Due to prevailing east winds, the runways are aligned east west, and frequently operates in an east flow with well-directed winds. This also works well for noise abatement, as the aircraft are naturally departing over the water, rather than over populated areas.
+
+During periods of high arrival volume, Miami utilizes land-and-hold-short operations (LAHSO) for arrivals on Runway 9 and Runway 12. Typically, Runway 9 arrivals keep the full length, and B737/A320 and similar (or smaller) aircraft land Runway 12 with instructions to hold short of Runway 9. This is most frequently the way the operation is run, though controllers have the ability to change it on the fly.
+
+During periods of high departure volume, Miami utilizes mostly Runway 8R, with Runway 9 being used for some heavy aircraft, assignment often determined based on parking position.
+
+MIA does not have any final monitor positions, and thus are unable to conduct simultaneous independent parallel approaches. Staggered separation must be used if parallel arrivals are in use.
+
+#### KFLL
+Fort Lauderdale / Hollywood International
+306,728 annual operations  
+18.2nm north-northeast of KMIA
+
+In terms of airspace, Fort Lauderdale is a very short distance away from Miami, and it is probably only for this reason that Fort Lauderdale does not have its own TRACON. It is a pretty busy place, and due to the proximity to another high volume airport (Miami), there is simply not a lot of room to get creative in the areas between these airports. It has two parallel east-west aligned runways (which roughly parallel Miami's), 10L and 10R. There is more general aviation traffic at KFLL than KMIA, but it is still overwhelmingly air carrier dominated (look to TMB and OPF for the GA activity).
+
+#### KTMB
+Miami Executive (Tamiami)
+282,066 annual operations
+11.8nm southwest of KMIA
+
+Almost all of Tamiami's traffic is itinerant general aviation traffic, which means the place can be a beehive of VFR during peak periods. Other than charters for the "top 1%", this airport exists solely to quench the people's thirst for a place to do pattern work, or to fly into and out of on the weekends. This is the kind of place for which "snowbirds" and "weekend warriors" get their names.
+
+#### KHWO
+North Perry
+190,955 annual operations
+12.6nm north-northeast of KMIA
+
+_to be completed_
```

Comment:
Are these to be completed before merging?

Reply:
filled in.

done